src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -348,9 +348,9 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
         /// Type of identity to be assigned to the server..
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "Type of Identity to be used. Possible values are SystemAsssigned, UserAssigned, SystemAssignedUserAssigned and None.")]
-        [PSArgumentCompleter("SystemAssigned", "UserAssigned", "SystemAssignedUserAssigned", "None")]
-        public string IdentityType { get; set; }
+            HelpMessage = "Type of Identity to be used. Possible values are SystemAssigned, UserAssigned, 'SystemAssigned,UserAssigned' and None.")]
+        [PSArgumentCompleter("SystemAssigned", "UserAssigned", "SystemAssigned,UserAssigned", "None")]
+        public Object[] IdentityType { get; set; }
```

Comment:
Object[] is not allowed in IdentityType either. Delete SystemAssignedUserAssigned will cause breaking change